pkg/helpers/version/version.go

```diff
@@ -136,6 +76,17 @@ func GetVersionBundle(version string) (VersionBundle, error) {
 		MulticlusterControlplane: "v0.3.0",
 	}
 
+	versionBundleList["0.13.0"] = VersionBundle{
+		Registration:             "v0.13.0",
+		Placement:                "v0.13.0",
+		Work:                     "v0.13.0",
+		Operator:                 "v0.13.0",
+		AddonManager:             "v0.13.0",
+		AppAddon:                 "v0.13.0",
+		PolicyAddon:              "v0.13.0",
```

Comment:
I don't see updated versions for any of the components (not the ones I checked).  Is the 0.13.0 release in the works now?

Reply:
it is in the progressing Operator/AddonManager have 0.13.0 already.